wiki/Modding/Kudosu/en.md

```diff
@@ -6,14 +6,17 @@ stub: true
 
 **Kudosu** is a reward system for [modding](/wiki/Modding). It awards points for successful mod posts in [beatmap discussions](/wiki/Beatmap_discussion). Accumulated points can be seen on a user's profile page.
 
+Kudosu is awarded to mod posts that are helpful and point out useful suggestions for the mapset. If a mapper believes that a mod post is useful and addresses a valid issue, they can click the thumbs up button and award kudosu to the user who posted the mod. 
+
 In [forum modding](/wiki/Modding/Forum_modding), kudosu was exchangeable for [star priority](/wiki/Modding/Star_priority).
```

Comment:
```suggestion
In [forum modding](/wiki/Modding/Forum_modding), kudosu used to be exchangeable for [star priority](/wiki/Modding/Star_priority).
```

Reply:
curious what u feel the difference is, these read identically to me

Reply:
slightly emphasises that it's not the case anymore